controller-manifests/elasticsearch-operator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+          verbs:
+          - get
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          - subjectaccessreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+      deployments:
+      - name: elasticsearch-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: elasticsearch-operator
+          template:
+            metadata:
+              labels:
+                name: elasticsearch-operator
+            spec:
+              serviceAccountName: elasticsearch-operator
+              containers:
+                - name: elasticsearch-operator
+                  image: quay.io/openshift/origin-elasticsearch-operator:latest
+                  imagePullPolicy: IfNotPresent
+                  command:
+                  - elasticsearch-operator
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.annotations['olm.targetNamespaces']
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: "elasticsearch-operator"
+                    - name: PROXY_IMAGE
+                      value: "quay.io/openshift/origin-oauth-proxy:latest"
+  version: 4.2.0
+  customresourcedefinitions:
+    owned:
+    - name: elasticsearches.logging.openshift.io
+      version: v1
+      kind: Elasticsearch
+      displayName: Elasticsearch
+      description: An Elasticsearch cluster instance
+      resources:
+      - kind: Deployment
+        version: v1
+      - kind: StatefulSet
+        version: v1
+      - kind: ReplicaSet
+        version: v1
+      - kind: Pod
+        version: v1
+      - kind: ConfigMap
+        version: v1
+      - kind: Service
+        version: v1
+      - kind: Route
+        version: v1
+      specDescriptors:
+      - description: Limits describes the minimum/maximum amount of compute resources required/allowed
+        displayName: Resource Requirements
+        path: nodeSpec.resources
+        x-descriptors:
+          - 'urn:alm:descriptor:com.tectonic.ui:resourceRequirements'
+      statusDescriptors:
+      - description: The current Status of the Elasticsearch Cluster
+        displayName: Status
+        path: cluster.status
+        x-descriptors:
+          - 'urn:alm:descriptor:io.kubernetes.phase'
+      - description: The number of Active Primary Shards for the Elasticsearch Cluster
+        displayName: Active Primary Shards
+        path: cluster.activePrimShards
```

Comment:
Consider: s/Prim/Primary/

Seems like 3 characters short is not worth shorting when below "Initializing" is kept in tact.